### DISCUSSION TITLE
How to close chrome after each scenario?

### DISCUSSION THREAD

**Discussion by jesus-medina1** [pytest-dev/pytest#9860] [Q&A] [answered]
Hello,
I am currently facing an issue in my framework where chrome does not close after each scenario. Instead, it loads in the same session. I would like chrome to close after each scenario, any thoughts on how I can fix my code to meet this?

conftest.py

<img width="458" alt="conftest" src="https://user-images.githubusercontent.com/102983710/163450726-abca9bc8-0dd1-462e-a643-b6a853ff857e.PNG">

test_login.py

<img width="458" alt="test_login" src="https://user-images.githubusercontent.com/102983710/163450811-5a43737d-8317-46db-ae71-5c3a9156a7e9.PNG">

**Comment by symonk** [comment, marked as answer]
Make your fixture `function` scoped

**Reply by jesus-medina1** [reply]
When changing the scope to function, I get this error
<img width="796" alt="error" src="https://user-images.githubusercontent.com/102983710/163460878-616cfbb7-692b-4b51-9b49-b77df998b5fe.PNG">
r

**Reply by jesus-medina1** [reply]
Nvm, forget to enter in autouse=True. This solved my error. thank you!
